### ISSUE TITLE
[FR] Using PFSbkM w/ UnRaid mapped drives

### ISSUE THREAD

**Issue by L0g1calMadn3ss** [GDX-X/PFS-BatchKit-Manager#44]
My computers do not have the storage space to handle all my games.  They're stored on my UnRaid server's mapped drives.  Anyway we can get PFSbkM to work with mapped drives?

**Comment by GDX-X** [comment]
I haven't tried but normally you can test and keep me informed

**Comment by L0g1calMadn3ss** [comment]
Sorry, I thought it went without saying that it doesn't work.

If you double-click the .bat in a mapped drive, nothing happens. If you open the .bat on the computer and then try to browse to the mapped drive letter it can't find the path. I've tried many variations of this, even with admin-cmd and terminal (which just tries to open cmd). Cmd either cannot find the mapped drive letter or cries about not being able to use unc paths (\\server\). I did try opening an admin-cmd, and then fully typing the unc path to the .bat (eg: \\server\!pfs...bat) and it opened PFSbkM but then just errored about unc paths thereafter.
